internal/service/mongodbemployeeaccessgrant/resource.go

```diff
@@ -89,6 +90,7 @@ func (r *rs) Delete(ctx context.Context, req resource.DeleteRequest, resp *resou
 }
 
 func (r *rs) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	conversion.ImportStateProjectIDClusterName(ctx, req, resp)
```

Comment:
one-line import implementation, ImportStateProjectIDClusterName can used by any other resource in TPF whose id is project_id and cluster_name